Bottom: 5cadb4f36c6d629cc3c64e97ccb9e358b78c6954
Top:    c1f41c6c5bd32517dbaec478e1cc0c1fc320008a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-12 01:42:50 +0100

Convert inline functions in hidgl.h to static inline

This agrees better with the mingw32 build for some reason.


---

diff --git a/src/hid/common/hidgl.h b/src/hid/common/hidgl.h
index b83e45b..31d28e8 100644
--- a/src/hid/common/hidgl.h
+++ b/src/hid/common/hidgl.h
@@ -48,7 +48,7 @@ void hidgl_flush_triangles (triangle_buffer *buffer);
 void hidgl_ensure_vertex_space (triangle_buffer *buffer, int count);
 void hidgl_ensure_triangle_space (triangle_buffer *buffer, int count);
 
-inline void
+static inline void
 hidgl_add_vertex_3D_tex (triangle_buffer *buffer,
                          GLfloat x, GLfloat y, GLfloat z,
                          GLfloat s, GLfloat t)
@@ -61,7 +61,7 @@ hidgl_add_vertex_3D_tex (triangle_buffer *buffer,
   buffer->vertex_count++;
 }
 
-inline void
+static inline void
 hidgl_add_vertex_tex (triangle_buffer *buffer,
                       GLfloat x, GLfloat y,
                       GLfloat s, GLfloat t)
